apps/angular/src/app/features/anime/anime-details-page/anime-details-page.component.html

```diff
@@ -1,10 +1,19 @@
 <div class="anime" *ngIf="anime$ | async as anime">
+	<div class="anime__actions">
+		<button mat-icon-button type="button" (click)="openDeleteConfirmationDialog()">
+			<mat-icon color="warn" aria-label="delete">delete</mat-icon>
+		</button>
+		<a routerLink="edit" mat-icon-button title="Anime edit page" type="button">
+			<mat-icon aria-label="edit">edit</mat-icon>
+		</a>
+		<a [routerLink]="['../create']" title="Anime create page" type="button" mat-raised-button>Create anime</a>
```

Comment:
According to semantics, the type attribute should be the `<button>` tag, not the `<a>` tag

```suggestion
		<a [routerLink]="['../create']" title="Anime create page" mat-raised-button>Create anime</a>
```